### PR TITLE
Add entities on dialogflow

### PR DESCRIPTION
**Pull request by linhofen**
(NOT FOR MERGE.)
I'd need a review for now.

### CODE REVIEW

**Review by Xahor — COMMENTED** [review]
From what i found in the documentation on dialogflow, everything looks good

File: test/test_dialogflow_service.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import unittest
+from api.dialogflow_service import DialogFlowService
+
+
+class MyTestCase(unittest.TestCase):
```
Comment (daesolus) [inline test/test_dialogflow_service.py:5]: Change the class name please.

File: tox.ini [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # content of: tox.ini , put in same dir as setup.py
 [tox]
-envlist = py36
+envlist = py36-32
```
Comment (daesolus) [inline tox.ini:3]: Why use 32 bits ? Our docker is  64 bits.

**Comment by daesolus** [comment]
This pull request has been merged directly into dialogflow_integration.

**Pull request closed without merging**